nfv_tempest_plugin/config.py

```diff
@@ -233,5 +233,10 @@
                 help='Configuration files'),
     cfg.BoolOpt('run_live_migration',
                 default=True,
-                help="Setup is able to run live migration")
+                help="Setup is able to run live migration"),
+    cfg.StrOpt('live_migration_mode',
+               default='block',
+               choices=['block', 'shared', 'none'],
```

Comment:
IMHO the option 'none' is redundant since it duplicates the meaning of 'run_live_migration'. And without the 'none' we could have kept the original line 152. 
If nevertheless we decide to keep the 'none' option than please note that the 'run_live_migration' is actually no longer needed. IIUC it was used only here in the whole plugin.

Reply:
Please note that _run_live_migration_ does not exist anymore, it has been replaced by _live_migration_mode_ and that's the reason why it has three possible values: 'block' for block migration, 'shared' for shared migration and 'none' if no live migration allowed. Also note the default value is 'block' to keep the current behavior and make possible to run the NUMA aware test by setting  _live_migration_mode_ to 'shared'.

Reply:
hmm, interesting, I am still seeing run_live_migration on line 234 if I open https://github.com/NFV18/nfv-tempest-plugin/pull/8/files or https://github.com/NFV18/nfv-tempest-plugin/commit/5f6830d022535f4c0219859f762ae64bccca102d
I thought the one that was removed was live_migration_block. 
Though it maybe I am missing something, I am still feeling uncomfortable after moving from gerrit to github

Reply:
You are absolutely right, I misunderstood you, my apologies. I forgot to delete _live_migration_mode_ from config.py which is unused (does nothing) now. Thanks.